charts/fdb-operator/templates/rbac/rbac_role.yaml

```diff
@@ -25,6 +25,14 @@ rules:
   - update
   - patch
   - delete
+- apiGroups:
+  - ""
+  resources:
+  - nodes
```

Comment:
This will only work if `.Values.globalMode.enabled` is set to true, otherwise this will be a `Role` but adding permissions to cluster wide resources require a `ClusterRole` (and `ClusterRolebinding`).

Reply:
Got it, will modify.